### DISCUSSION TITLE
[Android, Root Required] Exporting Images from Stocard

### DISCUSSION THREAD

**Discussion by Snuupy** [CatimaLoyalty/Android#309] [Show and tell]
If you are rooted, the way to export images is as follows:

1. go to `data/data/de.stocard/de.stocard.stocard/databases/sync_db`
2. copy that to your computer, load the file in DB Browser for SQLite
3. in sqlite browser, filter for your user ID
4. the images and entries will be available for export on the right side

From: https://github.com/TheLastProject/Catima/issues/223#issuecomment-891954148

**Comment by Mylou53** [comment]


**Reply by TheLastProject** [reply]
See the instructions Catima gives you when you choose to import from Stocard:
![image](https://user-images.githubusercontent.com/1885159/158254980-6e3255a8-33ea-4df0-b85e-d429225fcf9c.png)

You need to email support@stocardapp.com. They will most likely take 30 days to give you your export because that's how long the GDPR allows them to take (yup, Stocard goes out of their way to make leaving their app as annoying as possible). Do also note that the GDPR is an European law, so if you don't live in the EU Stocard may just refuse to give you your data.

**Comment by Snuupy** [comment]
If you aren't rooted you can always use an emulator and root that (ex. WSA - windows subsystem for Android + magisk on WSA) to grab that data

**Comment by hrenki** [comment]
If you still need to get those cards i made python gui to rescan your cards https://github.com/hrenki/stocard-extractor 
also managed to get sync_db without root on miui device

**Comment by svaningelgem** [comment]
Created a simple extractor to generate a zip file to import the data in Catima: https://gist.github.com/svaningelgem/3a4c4d886cf51f8f26642d260c7dadf1

Worked for me ;-)
